### ISSUE TITLE
path_ext_set does not recycle extension for vector

### ISSUE THREAD

**Issue by kiernann** [r-lib/fs#244]
If you want to create a vector of paths with the same extension.

``` r
fs::path_ext_set(LETTERS[1:3], "txt")
#> A.txt B.NA  C.NA
```

<sup>Created on 2020-02-20 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jennybc** [comment]
Probably a duplicate of #205. If true, then fixed already in #208.

**Comment by kiernann** [comment]
I've updated fs with `remotes::install_github("r-lib/fs")` and it seems fixed! Thanks.

``` r
packageVersion("fs")
#> [1] '1.3.1.9000'
fs::path_ext_set(LETTERS[1:3], "txt")
#> A.txt B.txt C.txt
```

<sup>Created on 2020-02-21 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
